src/vos/vos_ts.c

```diff
@@ -184,9 +140,7 @@ vos_ts_table_alloc(struct vos_ts_table **ts_tablep)
 		return -DER_NOMEM;
 
 	D_ALLOC_ARRAY(ts_table->tt_misses,
-		      (type_counts[VOS_TS_TYPE_CONT] * OBJ_MISS_SIZE) +
-		      (type_counts[VOS_TS_TYPE_OBJ] * DKEY_MISS_SIZE) +
-		      (type_counts[VOS_TS_TYPE_DKEY] * AKEY_MISS_SIZE));
+		      OBJ_MISS_SIZE + DKEY_MISS_SIZE + AKEY_MISS_SIZE);
```

Comment:
A bit nervous about the size of this array...

Reply:
I reduced it in latest patch.   The bottleneck I had was on the object level.   Now it's 64K entries at each level so about 22 M per xstream.